### PR TITLE
Add Atom feed support (#419)

### PR DESCRIPTION
**Pull request by ema-pe**
Hi, I've added Atom feed support for the website:

- There are two feeds, one located at `/it/news/feed.xml` and one at `/en/news/feed.xml`.
- I have also added feed autodiscovery, so websites and feed aggregator can retrieve automatically the feed position.

I am not an expert with frontend, so there is not a button on the website to get the feed. I was thinking to add a feed icon near Twitter and Medium:
Screenshot_2019-10-05 Developers Italia

But there is a problem: there is not a feed icon on [italia-icon-font](https://italia.github.io/bootstrap-italia/docs/utilities/icone/).

### CODE REVIEW

**Comment by ema-pe** [comment]
These are two Atom feed generated and [validated](https://validator.w3.org/feed/#validate_by_input): [Italian](https://github.com/italia/developers.italia.it/files/3693122/feed-en.txt) and [English](https://github.com/italia/developers.italia.it/files/3693123/feed-it.txt)

**Comment by ema-pe** [comment]
A solution is to add a custom SVG, this is my result:

Header
Footer

I do not know CSS therefore I am not capable to add necessary rules to adjust the icon with others, anyway in [this commit](https://github.com/ema-pe/developers.italia.it/commit/8bc7f122c03c4bf2ea6ff38b79377e63ec578dff) (on a separate branch) you can find the icon and the HTML node.

File: Gemfile.lock [hunk 1]
```diff
@@ -148,7 +148,7 @@ DEPENDENCIES
   tzinfo-data
 
 RUBY VERSION
-   ruby 2.6.0p0
+   ruby 2.6.3p62
```
Comment (libremente) [inline Gemfile.lock]: Why is this needed?

File: Gemfile.lock [hunk 2]
```diff
@@ -148,7 +148,7 @@ DEPENDENCIES
...
 
 BUNDLED WITH
-   2.0.1
+   2.0.2
```
Comment (libremente) [inline Gemfile.lock]: Same as before

File: Gemfile.lock (hunk 1 above)
Comment (ema-pe) [reply Gemfile.lock]: It is my local development environment. You can ignore this change.

File: Gemfile.lock (hunk 2 above)
Comment (ema-pe) [reply Gemfile.lock]: It is my local development environment. You can ignore this change. (same as before)

**Comment by ema-pe** [comment]
Now tests should work.

File: _layouts/feed.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  <subtitle>{{ site.data.l10n[page.locale].t.site_subtitle | xml_escape }}</subtitle>
+  <icon>{{ site.feed.icon }}</icon>
+  <logo>{{ site.feed.logo }}</logo>
+
+  <author>
+    <name>{{ site.author | xml_escape }}</name>
+  </author>
+
+  {%- assign posts = site.posts | where_exp:"post", "post.locale == page.locale" | limit:site.feed.limit -%}
+  {% for post in posts %}
```
Comment (sebbalex) [inline _layouts/feed.xml]: It would be nice to have in feed also software catalog, thoughts @libremente

Comment (libremente) [reply _layouts/feed.xml:21]: Yes @sebbalex that would be really nice!

**Comment by libremente** [comment]
Hi @ema-pe, sorry for the extreme delay, lately it has been hectic. I think it could be nice to merge this in the next sprint, I'll review it and let you know my toughts soon!  cc @sebbalex

**Comment by ema-pe** [comment]
Hi @libremente, I am not disappeared, I'm here!
I am also on Developers Italia Slack, you can reach me with this nickname `@ema-pe`

**Comment by libremente** [comment]
@ema-pe @sebbalex this is great and it's working as expected! I am just wondering what to do: we can merge this but could you please open a new PR with this https://github.com/ema-pe/developers.italia.it/commit/8bc7f122c03c4bf2ea6ff38b79377e63ec578dff? So we can make the RSS feed more discoverable also by humans. What do you think? Thanks!

**Comment by sebbalex** [comment]
good catch @libremente

**Comment by ema-pe** [comment]
@libremente  @sebbalex 

Yes, I can fix the problems, but I'm very busy now, I can try to fix in this or next weekend! Is there a deadline?

The problem with [this commit](https://github.com/ema-pe/developers.italia.it/commit/8bc7f122c03c4bf2ea6ff38b79377e63ec578dff) is that there is not a feed icon in [italia-icon-font](https://italia.github.io/bootstrap-italia/docs/utilities/icone/), so I've only downloaded a simple SVG from the Web. I don't know really much CSS, but I can try to improve it.

**Comment by sebbalex** [comment]
> Yes, I can fix the problems, but I'm very busy now, I can try to fix in this or next weekend! Is there a deadline?

Thanks a lot for the time you are spending on this. We work in weekly sprint and since this is not yet tracked I can move it to next sprint.

> The problem with this commit is that there is not a feed icon in italia-icon-font, so I've only downloaded a simple SVG from the Web. I don't know really much CSS, but I can try to improve it.

Yes, you are right, keep this commit in, we will size it correctly in a dedicated issue.

Thanks.

**Comment by sebbalex** [comment]
@ema-pe any news on that?

**Comment by ema-pe** [comment]
> @ema-pe any news on that?

Hi, unfortunately there are no news, I'm busy all time with university. I have the last exam on 13/07, then I can have some free time to help, but I can't guarantee.

But I think it is better to open a new pull request, this is too old.

**Comment by bfabio** [comment]
Opened #643. Thanks @ema-pe!
